### ISSUE TITLE
RemovedInSphinx17Warning

### ISSUE THREAD

**Issue by csams** [spatialaudio/nbsphinx#127]
When generating html I get the following:
```
...
generating indices... genindex py-modindex
/home/csams/git_repos/insights-core/lib/python2.7/site-packages/sphinx/application.py:445: RemovedInSphinx17Warning: app.status_iterator() is now deprecated. Use sphinx.util.status_iterator() instead.
  RemovedInSphinx17Warning)
```
Using the following:
```
nbsphinx==0.2.14
Sphinx==1.6.4
```

The problem is at line 1177 of `nbsphinx.py`.

**Comment by csams** [comment]
Apologies - I see that this is fixed in master. Maybe it just hasn't been updated in pypi.

**Comment by csams** [comment]
When is another release planned?

**Comment by mgeier** [comment]
I'm currently on the road, I can make a new release in 1 week, probably a
few days later.

I was waiting for confirmation on
https://github.com/spatialaudio/nbsphinx/pull/123, which I would like to
include in the next release.

To speed up things, can you please check if that works for you?

**Comment by mgeier** [comment]
For reference, this was fixed in #115.

I've just created a new release (https://pypi.python.org/pypi/nbsphinx/0.2.15) which should solve the problem.
